components/post_view/post_list/post_list.jsx

```diff
@@ -0,0 +1,215 @@
+                    getPostList={this.getPostList}
+                />
+            );
+
+            const currentPostDay = Utils.getDateForUnixTicks(post.create_at);
+            if (currentPostDay.toDateString() !== previousPostDay.toDateString()) {
+                postCtls.push(
+                    <DateSeparator
+                        key={currentPostDay.getTime()}
+                        date={currentPostDay}
+                    />
+                );
+            }
+
+            const isNotCurrentUser = post.user_id !== currentUserId || isFromWebhook(post);
+            if (isNotCurrentUser &&
+                    lastViewed !== 0 &&
+                    post.create_at > lastViewed &&
+                    !Utils.isPostEphemeral(post) &&
+                    !renderedLastViewed) {
+                renderedLastViewed = true;
+
+                // Temporary fix to solve ie11 rendering issue
+                let newSeparatorId = '';
+                if (!UserAgent.isInternetExplorer()) {
+                    newSeparatorId = 'new_message_' + post.id;
+                }
+                postCtls.push(
+                    <div
+                        id={newSeparatorId}
+                        key='unviewed'
+                        ref='newMessageSeparator'
+                        className='new-separator'
+                    >
+                        <hr
+                            className='separator__hr'
+                        />
+                        <div className='separator__text'>
+                            <FormattedMessage
+                                id='posts_view.newMsg'
+                                defaultMessage='New Messages'
+                            />
+                        </div>
+                    </div>
+                );
+            }
+
+            postCtls.push(postCtl);
+            previousPostDay = currentPostDay;
+        }
+
+        return postCtls;
+    }
+
+    getPostList = () => {
+        return this.refs.postlist;
+    }
+
+    loadOlderPosts = () => {
+        const oldestPost = this.props.posts[this.props.posts.length - 1];
+        let oldestPostId = oldestPost.id;
+        if (oldestPost.type === Posts.POST_TYPES.COMBINED_USER_ACTIVITY) {
+            oldestPostId = oldestPost.system_post_ids[0];
+        }
+        this.props.actions.loadOlderPosts(oldestPostId);
+    }
+
+    loadNewerPosts = () => {
+        const newestPost = this.props.posts[0];
+        let newestPostId = newestPost.id;
+        if (newestPost.type === Posts.POST_TYPES.COMBINED_USER_ACTIVITY) {
+            newestPostId = newestPost.system_post_ids[0];
+        }
+        this.props.actions.loadNewerPosts(newestPostId);
+    }
+
+    render() {
+        const posts = this.props.posts;
+
+        let topRow;
+        let bottomRow;
+
+        if (!this.props.olderPosts.allLoaded && !this.props.olderPosts.loading && !this.props.disableLoadingPosts) {
```

Comment:
PostList is an abstraction to use it in different places later on. something like RHS. This still needs to have scroll changes so the component might not be all that simple as it looks now.

Footer and Header in postList wrapper are related to the parent component and can be different for different postList. I.e RHS might not have the same header and footer. 
footer and header here in this component are independent of the component that they are rendered in. They either show load more or a loader in case of infinite loading are generic

Does virtualisation effect the header if it is outside of the virtualized post_list component?